pythonPackages.nix

```diff
@@ -7,4 +7,12 @@ in {
   pyscf = callPackage ./pyscf { };
   pyquante = callPackage ./pyquante { };
   pychemps2 = callPackage ./chemps2/PyChemMPS2.nix { };
+  qcelemental = callPackage ./qcelemental { };
+  qcengine = callPackage ./qcengine { };
+  gau2grid-1_3_1 = callPackage ./gau2grid/1_3_1.nix { };
```

Comment:
As a general comment for the cases, where we have multiple versions of a package: as far as I can see, the derivations are identical except for `version` and `sha256`. I would be better to just have one derivation file instead and make `version` and `sha256` input parameters, with defaults for the corresponding parameters for the latest stable (I know should do the same for all my molpro derivations).

Reply:
Has been done.